modin/config/pubsub.py

```diff
@@ -74,6 +74,9 @@ class Parameter(object):
     type = str
     default = None
     is_abstract = True
+    # This flag is used to detect a way of setting value for an entity.
+    # 0 - default value, 1 - value from config source, 2 - value set by a user.
```

Comment:
I would rather we have these as constant values, not inside a comment

Reply:
Yes, it makes sense. Fixed.